frontend/src/app/components/Contact/Contact.tsx

```diff
@@ -0,0 +1,54 @@
+import styles from "@/app/components/Sidebar/sidebar.module.scss";
+import matter from "gray-matter";
+import ReactMarkdown from "react-markdown";
+import fs from "fs/promises";
+import path from "path";
+
+async function getContactMarkdown() {
+    try {
+        const filePath = path.join(process.cwd(), "content", "contact.md");
```

Comment:
The current working directory differs between production and development. It would be better to use the `HOME` environment variable to locate the file. This makes it work in production:

```suggestion
        const filePath = path.join(process.env.HOME || "", "frontend", "content", "contact.md");
```